src/test/java/seedu/momentum/model/timer/WorkDurationTest.java

```diff
@@ -0,0 +1,64 @@
+package seedu.momentum.model.project;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.time.LocalDateTime;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.momentum.model.timer.Time;
+import seedu.momentum.model.timer.WorkDuration;
+import seedu.momentum.testutil.TypicalWorkDuration;
+
+public class WorkDurationTest {
+
+    @Test
+    public void isSameDuration() {
+        // different phone and email -> returns false
+        //        new ProjectBuilder(ALICE).withPhone(VALID_PHONE_BOB).withEmail(VALID_EMAIL_BOB).build();
+        //        assertFalse(ALICE.isSameProject(editedAlice));
+
+        // different name -> returns false
+        //        Project editedAlice = editedAlice = new ProjectBuilder(ALICE).withName(VALID_NAME_BOB).build();
+        //        assertFalse(ALICE.isSameProject(editedAlice));
+
+        // same name, same phone, different attributes -> returns true
+        //        editedAlice = new ProjectBuilder(ALICE).withEmail(VALID_EMAIL_BOB)
+        //                .withTags(VALID_TAG_HUSBAND).build();
+        //        assertTrue(ALICE.isSameProject(editedAlice));
+
+        // same name, same email, different attributes -> returns true
+        //        editedAlice = new ProjectBuilder(ALICE).withPhone(VALID_PHONE_BOB)
+        //                .withTags(VALID_TAG_HUSBAND).build();
+        //        assertTrue(ALICE.isSameProject(editedAlice));
+
+        // same name, same phone, same email, different attributes -> returns true
+        //        editedAlice = new ProjectBuilder(ALICE).withTags(VALID_TAG_HUSBAND).build();
+        //        assertTrue(ALICE.isSameProject(editedAlice));
+
+        // same values -> returns true
+        WorkDuration durationCopy = new WorkDuration(TypicalWorkDuration.DURATION_ONE_DAY.getStartTime(),
+                TypicalWorkDuration.DURATION_ONE_DAY.getStopTime());
+        assertTrue(TypicalWorkDuration.DURATION_ONE_DAY.isSameDuration(durationCopy));
+
+        // same object -> returns true
+        assertTrue(TypicalWorkDuration.DURATION_ONE_DAY.isSameDuration(TypicalWorkDuration.DURATION_ONE_DAY));
+
+        // null -> returns false
+        assertFalse(TypicalWorkDuration.DURATION_ONE_DAY.isSameDuration(null));
+
+        // different project -> returns false
```

Comment:
Not sure if I understood your TypicalWorkDuration class fully, but is it comparing different projects here?

Reply:
TypicalWorkDuraiton is just a class to provide dummy data for testing. I defined some simple durations like 1 day, week, month, etc for testing